charts/console/templates/console/secret-credentials.yaml

```diff
@@ -23,4 +23,8 @@ data:
   CDK_ADMIN_PASSWORD: {{ include "common.secrets.lookup" (dict "secret" $secretName "key" "CDK_ADMIN_PASSWORD" "defaultValue" $adminPassword "context" $) }}
   CDK_DATABASE_PASSWORD: {{ required "config.database.password MUST be set in values" .Values.config.database.password | b64enc }}
   CDK_DATABASE_USERNAME: {{ required "config.database.username MUST be set in values" .Values.config.database.username | b64enc }}
+  {{- if .Values.config.conduktorSql.enabled }}
+    CDK_SQL_DATABASE_PASSWORD: {{ required "config.conduktorSql.password MUST be set in values" .Values.config.conduktorSql.password | b64enc }}
+    CDK_SQL_DATABASE_USERNAME: {{ required "config.conduktorSql.username MUST be set in values" .Values.config.conduktorSql.username | b64enc }}
```

Comment:
correct env var are
`CDK_KAFKASQL_DATABASE_USERNAME` and `CDK_KAFKASQL_DATABASE_PASSWORD`